Un 211 continued editor feedback

William-Blackie opened the pull request:
Ticket URL: : https://national-archives.atlassian.net/browse/UN-496
## About these changes

Changes requested: https://national-archives.atlassian.net/browse/UN-496?focusedCommentId=42916

## How to check these changes
Load a record article page with the transcirption gallery, images are now the original sizes. After speaking with the content team they will take ownership of sizes / layout changes and want the abillity to use large images.


## Before assigning to reviewer, please make sure you have

- [ ] Checked things thoroughly before handing over to reviewer.
- [ ] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [ ] Ensured that PR includes only commits relevant to the ticket.
- [ ] Waited for all CI jobs to pass before requesting a review.
- [ ] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

William-Blackie commented on etna/records/fields.py:
```diff
@@ -18,12 +18,13 @@ class LazyRecord(SimpleLazyObject):
 
         def _fetch_record():
-            return Record.api.fetch(iaid=iaid)
+            return self.records_client.fetch(iaid=iaid)
```
It would be great to get your thoughts on these changes @ababic. Currently when loading the Record model no longer has the `api` method. I'm not 100% clear on the changes to this model underneath so any clarification would be appreciated!

William-Blackie commented on templates/includes/record-revealed.html:
```diff
@@ -7,10 +7,10 @@
                     {% image page.teaser_image fill-390x390 as teaser_image %}
-                    <img height=390 width=390 src="{{ teaser_image.url }}" alt="" />
+                    <img height=390 width=390 src="{{ teaser_image.url }}" alt="" class="explore-intro__image" />
                 {% endif %}
```
This makes the intro image a little more flexible so it's not causing layout shift.

JohnHeeryTNA commented:
Happy to approve the FE pieces, however there seem to be some merge conflicts, could you have a look please @William-Blackie

William-Blackie commented:
> Happy to approve the FE pieces, however there seem to be some merge conflicts, could you have a look please @William-Blackie

Hey @JohnHeeryTNA, I've rebased and removed the conflicting files / pulled in the changes which actually did the work I was. Could you mark as approved?

JohnHeeryTNA reviewed (approved):
approving changes
